patch/0001-v4.19-6-Mellanox-platform-Backport-patches-for-new-M.patch

```diff
@@ -40,39 +41,38 @@ It contains backport patches from v5.* kernels:
 Signed-off-by: Vadim Pasternak <vadimp@mellanox.com>
 Signed-off-by: Stephen Sun <stephens@mellanox.com>
 ---
- drivers/hwmon/mlxreg-fan.c                         |   90 +-
```

Comment:
please verify this patch is ok. since there are a few rejection when applying the original patch to 4.19.118.